### PR TITLE
2.1 changelog

### PR DESCRIPTION
**Pull request by dstansby**
The first commit is the one that needs cherry picking into the `master` branch, as it gets rid of the individual changelog files. The second commit is some copy editing, the third commit is fixing links.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,402 @@
+sunpy 3.0.0 ()
```
Comment (nabobalis) [inline CHANGELOG.rst:1]:
```suggestion
sunpy 2.1.0 ()
```

**Comment by nabobalis** [comment]
Should we not just cherry pick the entire PR?

**Comment by dstansby** [comment]
> Should we not just cherry pick the entire PR?

Oh maybe, yeah.

File: CHANGELOG.rst (hunk 1 above)
Comment (dstansby) [reply CHANGELOG.rst:1]: lol woops

**Comment by nabobalis** [comment]
> > Should we not just cherry pick the entire PR?
> 
> Oh maybe, yeah.

I feel like if we copy edit the changelog and fix the links, you'll need to do that on master eventually.

**Pull request merged**